packages/apollo-cache-control/jest.config.js

```diff
@@ -0,0 +1,3 @@
+const config = require('../../jest.config.base');
+
+module.exports = Object.assign({}, config);
```

Comment:
Slight preference for `Object.assign(Object.create, config)` here (and in general) to create an object that doesn't extend `Object.prototype`, just because of the potential for conflicting `prototype` properties. (The type of thing that surfaces as a very bizarre edge case).

Reply:
Ah, yes, that makes sense. I'd prefer to switch to object spreads when we drop support for Node 6 however. Not sure how those deal with `Object.prototype`.